Userland/Games/2048/Game.cpp

```diff
@@ -198,6 +219,72 @@ Game::MoveOutcome Game::attempt_move(Direction direction)
     return MoveOutcome::InvalidMove;
 }
 
+void Game::add_evil_tile()
+{
+    size_t worst_row = 0;
+    size_t worst_column = 0;
+    u32 worst_value = 2;
+
+    size_t most_free_cells = m_grid_size * m_grid_size; // a big value to min() against
+
+    // Largest theoretically possible score after a move.
+    // All the cells are filled with the same tile, so after a move every tile merges.
+    size_t worst_score = m_score + largest_tile() * m_grid_size * m_grid_size;
```

Comment:
You're overthinking this; just use the maximum value of `size_t` for `most_free_cells` and `worst_score`.